glTF-Toolkit/glTF-Toolkit.vcxproj

```diff
@@ -148,8 +148,7 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
     <Lib>
-      <AdditionalDependencies>
-      </AdditionalDependencies>
+      <AdditionalDependencies>shlwapi.lib</AdditionalDependencies>

```

Comment:
Please remove - this doesn't work with UWP

Reply:
ok